trunk/Admin/partials/loop-post-types.php

```diff
@@ -4,9 +4,13 @@
                 href="<?php echo admin_url( 'post-new.php' ) . '?post_type=' . $postType->name; ?>"></a>
     </div>
     <div class="admin-quickbar-postlist-inner">
-        <?php
-        // loop categories of current post-type
-        echo $categoryLoop;
-        ?>
+        <?php foreach ($categoryLoop as $categoryName => $renderedPosts) : ?>
```

Comment:
Rethink the naming of `$categoryLoop`. It does not contain a loop.

Reply:
Renamed to $postsByCategory